### PR TITLE
data: dineshlalam15

### PR DESCRIPTION
**Pull request by dineshlalam15**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5706"></a>

### CODE REVIEW

File: data/dineshlalam15.json [hunk 1]
```diff
@@ -13,5 +13,15 @@
       "url": "https://github.com/dineshlalam15",
       "icon": "FaGithub"
     }
+    {
+      "name": "Hey! Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/dinesh-kumar-lalam-2584a8207/",
+      "icon": "FaLinkedIn"
+    }
```
Comment (kumarsonsoff3) [inline data/dineshlalam15.json:20]: Please add a missing comma `,` here.

```suggestion
    },
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/dineshlalam15.json [hunk 2]
```diff
@@ -17,7 +17,7 @@
       "name": "Hey! Let's connect on LinkedIn",
       "url": "https://www.linkedin.com/in/dinesh-kumar-lalam-2584a8207/",
       "icon": "FaLinkedIn"
```
Comment (khushalarora) [inline data/dineshlalam15.json:19]: Hi @dineshlalam15, thank you for contributing  :tada:
Please replace it with &nbsp; **' FaLinkedin '** &nbsp; :+1: 
```Suggestion
      "icon": "FaLinkedin"
```

File: data/dineshlalam15.json [hunk 3]
```diff
@@ -13,5 +13,15 @@
       "url": "https://github.com/dineshlalam15",
       "icon": "FaGithub"
     }
```
Comment (kumarsonsoff3) [inline data/dineshlalam15.json:15]: There is another missing comma `,`

```suggestion
    },
```

**Review by kumarsonsoff3 — CHANGES_REQUESTED** [review]

File: data/dineshlalam15.json [hunk 4]
```diff
@@ -12,6 +12,16 @@
...
+    },
+    {
+      "name": "Hey! Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/dinesh-kumar-lalam-2584a8207/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Look into my blogs here :)",
+      "url": "https://hashnode.com/@dineshlalam15",
+      "icon": "FaHashnode"
```
Comment (kumarsonsoff3) [inline data/dineshlalam15.json:24]: You probably be using an invalid icon here, it should be `SiHashnode`.

```suggestion
      "icon": "SiHashnode"
```

**Pull request merged**